Katello/katello
# Fixes #37552 - container push repo content view support

#### What are the changes introduced in this pull request?
- Allows publishing container push repositories in content views
- Allows deletion of container push repositories
- Adds authorization checks for container push
- Skips metadata generation for container push repositories
- Adds translations for container push errors

#### Considerations taken when implementing this change?
Other update actions on container push repositories will cause weirdness. I considered fixing that here too, but I think it'll be better for another PR. We'll need to change up the repo edit UI for container push repos.

#### What are the testing steps for this pull request?
1) Push some container repos to Katello and publish them in a content view
2) Try filtering content with CV publishes
3) Try deleting container push repos
4) Try pushing repositories as a user that shouldn't be able to (perhaps only has ability to create personal access tokens)
5) Try regenerating metadata for container push repositories -- see that nothing fails
